### PR TITLE
Feature/organization

### PR DESCRIPTION
**Pull request by garyluu**
Hook up the register organization component to the organizations page.
Hook up the organization component to the organizations page

Add functionality and integration tests for CRU organization (without description markdown) and CRU collection (without entries)

Added hammerjs to get rid of warning (could possibly revert)

https://dashboard.cypress.io/#/projects/1ya4vj/runs/4889/specs organization.ts video to see it

### CODE REVIEW

**Comment by coverbeck** [comment]
+1 on Cypress video --great way to show

File: src/app/organizations/state/register-organization.service.ts [hunk 1]
```diff
@@ -68,4 +155,43 @@ export class RegisterOrganizationService {
...
+      const editedOrganization: Organisation = {
+        name: organizationFormState.name,
+        topic: organizationFormState.topic,
+        link: organizationFormState.link,
+        location: organizationFormState.location,
+        email: organizationFormState.contactEmail,
+        // Setting approved to true to appease compiler.  Webservice should completely ignore this.
+        approved: true
+      };
+      this.alertService.start('Editing organization');
```
Comment (coverbeck) [inline src/app/organizations/state/register-organization.service.ts]: I would say `Updating organization` -- Editing happens in the UI

File: src/app/organizations/collections/state/create-collection.service.ts [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+      description: collectionFormState.description
+    };
+    const organizationID = this.organizationQuery.getSnapshot().organization.id;
+    this.createCollectionStore.setLoading(true);
+    this.organisationsService.createCollection(organizationID, collection).pipe(
+      finalize(() => this.createCollectionStore.setLoading(false)))
+      .subscribe((newCollection: Collection) => {
+        this.createCollectionStore.setError(false);
+        this.matDialog.closeAll();
+        this.matSnackBar.open('Creating collection successful');
```
Comment (coverbeck) [inline src/app/organizations/collections/state/create-collection.service.ts]: Is there a reason that you're not using alert service here and on line 56?

File: src/app/organizations/collections/collections.component.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    <ng-template #hasCollections>
+      <div fxLayout="row wrap" fxLayoutAlign="space-evenly stretch">
+        <mat-card fxFlex="400px" *ngFor="let collection of ((collections$ | async) | keyvalue)" class="m-1">
+          <mat-card-header>
+            <mat-card-title>{{collection.value.name}}</mat-card-title>
+            <mat-card-subtitle>{{collection.value.description}}</mat-card-subtitle>
+          </mat-card-header>
+          <mat-card-content>
+            <p>
+              The Shiba Inu is the smallest of the six original and distinct spitz breeds of dog from Japan.
```
[the remark this answers is not in the document]
Comment (garyluu) [reply src/app/organizations/collections/collections.component.html]: Lots of them are placeholders until it's been implemented

File: src/app/organizations/state/register-organization.service.ts (hunk 1 above)
Comment (garyluu) [reply src/app/organizations/state/register-organization.service.ts]: :ok_hand:

**Pull request merged**